pkg/harvester/l10n/en-us.yaml

```diff
@@ -1360,6 +1360,7 @@ advancedSettings:
     'harv-default-vm-termination-grace-period-seconds': Config the VM termination grace period for VM stop.
     'harv-ntp-servers': Configure NTP server. You can configure multiple IPv4 addresses or host addresses.
     'harv-auto-rotate-rke2-certs': The certificate rotation mechanism relies on Rancher. Harvester will automatically update certificates generation to trigger rotation.
+    'harv-kubeconfig-default-token-ttl-minutes': 'TTL (in minutes) applied on all kubeconfig tokens. When set to 0, the token never expires.'
```

Comment:
Any comments on a better description are welcome to reflect [this](https://github.com/harvester/harvester/pull/5896#issuecomment-2149667868).

Reply:
:rofl:

Reply:
@ibrokethecloud

Reply:
sounds good to me

Reply:
`TTL (in minutes) applied on harvester admin kubeconfig files. Default is 0 mins, which means to never expire`?

Reply:
> `TTL (in minutes) applied on harvester admin kubeconfig files. Default is 0 mins, which means to never expire`?

I have used your suggestion in a slightly modified form. Thx to @w13915984028 for the translation.